pandas/tests/frame/test_indexing.py

```diff
@@ -364,6 +364,7 @@ def test_getitem_ix_mixed_integer(self):
         assert_frame_equal(result, expected)
 
         with catch_warnings(record=True):
+            simplefilter("ignore", DeprecationWarning)
```

Comment:
Same comment as before but just wonder if perhaps the `pytest.mark.filterwarnings` decorator could give flexibility to apply at function level to minimize diff